.ebextensions/04_create_apache_conf.config

```diff
@@ -17,9 +17,9 @@ files:
       # This file was overwritten during deployment
       # by .ebextensions/05_overwrite_apache_conf.config
 
-      Listen 80
+      Listen 8080
 
-      <VirtualHost *:80>
+      <VirtualHost *:8080>
```

Comment:
Elastic Beanstalk _appears_ to be serving requests over port 80; but the instances themselves are using NAT tables to route from 80 to 8080; why?; because!